marshal.go

```diff
@@ -355,6 +390,7 @@ func (x *GoSNMP) send(packetOut *SnmpPacket, wait bool) (result *SnmpPacket, err
 		return nil, fmt.Errorf("&GoSNMP.Conn is missing. Provide a connection or use Connect()")
 	}
 
+	// detected unknown engine id error,
```

Comment:
This comment doesn't seem related, copy n paste error?

Reply:
thx, removed